proto-lens-tests/proto-lens-tests.cabal

```diff
@@ -162,6 +162,7 @@ Test-Suite text_format_test
                , proto-lens
                , proto-lens-protoc
                , proto-lens-tests
+               , proto-lens-protobuf-types
```

Comment:
You'll need to add this to the dependencies of the *library*, not the test.  Sorry for this confusing behavior; see also the comment at l.29.